### ISSUE TITLE
Run error in hyak

### ISSUE THREAD

**Issue by YifanCheng** [UW-Hydro/MetSim#60]
Problem description:

- When I ran the example code given in Metsim, it reported following error message
```
  File "/usr/lusers/yifanc6/.conda/envs/metsim/lib/python3.5/site-packages/numba/decorators.py", line 169, in wrapper
    disp.enable_caching()
  File "/usr/lusers/yifanc6/.conda/envs/metsim/lib/python3.5/site-packages/numba/dispatcher.py", line 469, in enable_caching
    self._cache = FunctionCache(self.py_func)
  File "/usr/lusers/yifanc6/.conda/envs/metsim/lib/python3.5/site-packages/numba/caching.py", line 569, in __init__
    self._impl = self._impl_class(py_func)
  File "/usr/lusers/yifanc6/.conda/envs/metsim/lib/python3.5/site-packages/numba/caching.py", line 305, in __init__
    "for file %r" % (qualname, source_path))
RuntimeError: cannot cache function 'solar_geom': no locator available for file '/usr/lusers/yifanc6/.conda/envs/metsim/lib/python3.5/site-packages/metsim-0.0.0-py3.5.egg/metsim/physics.py'
```

Environments:

- I created a virtual environment using `environments.yml` file

Modules:
  1) modules                     4) icc_14.0.2                  7) epel_packages
  2) canopy_1.5.4                5) netcdf_4.3.1.1-icc_14.0.2   8) git_2.4.4
  3) anaconda_2.4                6) contrib/hydro

PS: This problem was solved by deleting `cache = True` in line 173 in physics.py by Andrew but this is not a good way to solve this problem

**Comment by arbennett** [comment]
I did some digging on this, and couldn't find anything about how to work around it.  I think we are going to have to remove caching for the JIT.  The main downside to this is that it adds a few seconds to startup while numba compiles the relevant functions.

**Comment by bartnijssen** [comment]
@arbennett : Was this fixed before V0.1 was released as I had requested?

**Comment by arbennett** [comment]
@bartnijssen this was not in, but has been solved in develop now.
